chains.yaml

```
@@ -295,6 +295,18 @@
   build-env:
     - BUILD_TAGS=muslc
 
+# Duality
+- name: duality
+  github-organization: duality-labs
+  github-repo: duality
+  language: go
+  build-target: install
```

After pulling in main:

```suggestion
  build-target: make install
```